snippets/service-account-auto-secret-removed.adoc

```diff
@@ -0,0 +1,17 @@
+// When including this file, ensure that {FeatureName} is set immediately before
+// the include. Otherwise it will result in an incorrect replacement.
+
+[id="auto-generated-sa-token-secrets_{context}"]
+== About automatically-generated service account token secrets
+
+In {product-version}, {product-title} is adopting an link:https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG/CHANGELOG-1.24.md#urgent-upgrade-notes-1[enhancement from upstream Kubernetes] where the `LegacyServiceAccountTokenNoAutoGeneration` feature is enabled by default. As a result, when creating new serivce accounts (SA), a service account token secret is no longer automatically genereated. Previously, {product-title} automatically added a service account token to a secret for each new SA. 
+
+However, some features and workloads need service account token secrets to communicate with the Kubernetes API server, for example, the OpenShift Controller Manager. While this requirement will be changed in a future release, it remains in {product-title} {product-version}. As a result, if you need a service account token secret, you must manually use the TokenRequest API to request bound service account tokens or create a service account token secret. 
```

Comment:
> use the TokenRequest API to request
New command `oc create token` lands in 4.11 oc, it implements the request for the bound sa tokens using TokenRequest API. How about mentioning it somewhere to let customers know?

Reply:
@mburke5678 saw this PR is merged, what about this suggestion?

Reply:
@xingxingxia It appears this is a larger change, as this parameter is not documented yet, we would need to add a new section for it. Or, replace the existing docs we have on using the TokenRequest API. I do not know what the plan is.

Reply:
@xingxingxia I can follow up with you after GA to see if there are further doc updates we need to do for the new `oc create token` command. Would you mind opening a Jira with your thoughts and assigning it to me?

As for announcing it, we do at least mention it in this section: https://docs.openshift.com/container-platform/4.11/release_notes/ocp-4-11-release-notes.html#ocp-4-11-oc-commands-flags-tokens-deprecated. i also checked with @soltysh this morning (see [slack thread](https://coreos.slack.com/archives/GK58XC2G2/p1659361010892829)) to see if it needs its own announcement in the release notes, and he didn't think it was necessary.

Reply:
@xingxingxia also, can you also see if we need to make any updates to this section, about having to manually create the token now? https://docs.openshift.com/container-platform/4.11/authentication/understanding-and-creating-service-accounts.html#service-accounts-overview_understanding-service-accounts

Reply:
My suggestion is, need not add too much things, just mention it somewhere in document to let customers know (not only in RN such that onward releases include it). For example, just mention:
```
you must manually use the TokenRequest API to request bound service account tokens or create a service account token secret. You can also refer to `oc create token -h` command to see how to use CLI to request bound serviceaccount tokens.
```